WordPress/Classes/Utility/BuildInformation/RemoteFeatureFlag.swift

```diff
@@ -114,7 +138,7 @@ enum RemoteFeatureFlag: Int, CaseIterable {
         case .domainManagement:
             return "domain_management"
         case .dynamicDashboardCards:
-            return "dynamic_dashboard_cards"
+            return "domain_management_announcement_card"
```

Comment:
I've updated this one to match the Domain Management dynamic remote feature flag key.